tests/conftest.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 import ansible_runner
-import pkg_resources
+from importlib.metadata import version, PackageNotFoundError
```

Comment:
When you use `importlib-metadata` backport package, you need to import from `importlib_metadata` instead, so:
```suggestion
try:
    from importlib.metadata import version, PackageNotFoundError
except ModuleNotFoundError:
    from importlib_metadata import version, PackageNotFoundError
```